tensorboard/logo/BUILD

```diff
@@ -0,0 +1,19 @@
+# Description:
+# TensorBoard logo files.
+
+package(default_visibility = ["//tensorboard:internal"])
+
+licenses(["notice"])  # Apache 2.0
+
+exports_files(["LICENSE"])
+
+sh_binary(
+    name = "inline_favicon",
+    srcs = ["inline_favicon.sh"],
+    data = [
+        "favicon.png",
+    ],
+    visibility = [
+        "//tensorboard:internal",
+    ],
```

Comment:
redundant since we have the default visibility on L4.

Reply:
Removed; thanks.